gomock/matchers_test.go

```diff
@@ -142,3 +142,176 @@ func TestAssignableToTypeOfMatcher(t *testing.T) {
 		t.Errorf(`AssignableToTypeOf(context.Context) should not match ctxWithValue`)
 	}
 }
+
+func TestInAnyOrder(t *testing.T) {
+	tests := []struct {
+		name      string
+		wanted    interface{}
+		given     interface{}
+		wantMatch bool
+	}{
+		{
+			name:      "match_for_equal_slices",
```

Comment:
nit: can use spaces instead of underscores for test names.

Reply:
Sure, it's just nice to be able to copy name of a test with underscores directly from gotest log as is and search for it in codebase. :)

Reply:
Looks like this was not resolved.